WasatchNET/WasatchNET.csproj

```diff
@@ -117,35 +117,28 @@
     <Compile Include="AcquisitionStatus.cs" />
     <Compile Include="AndorEEPROM.cs" />
     <Compile Include="AndorSpectrometer.cs" />
-    <Compile Include="BoulderEEPROM.cs" />

```

Comment:
Technically, if you removed these lines from the .csproj, I'm not sure you needed to actually delete the source code files, but okay.